.github/workflows/build-notebooks-TEMPLATE.yaml

```
@@ -0,0 +1,703 @@
+      - name: Show error logs (on failure)
+        if: ${{ failure() }}
+        run: |
+          set -Eeuxo pipefail
+
+          journalctl -xe
+          ls -AlF /var/run/podman/podman.sock || echo "Socket /var/run/podman/podman.sock not found"
+          sudo ss -xlpn | grep 'podman.sock' || echo "No active listener found for podman.sock via ss"
+
+      - name: Calculate image name and tag
+        id: calculated_vars
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2036:warning:4:1: If you wanted to assign the output of the pipeline, use a=$(b | c) 

(shellcheck)